packages/components/src/components/carousel/carousel-align-header.ts

```diff
@@ -0,0 +1,2 @@
+export const HEADER_ALIGNS = ['left', 'center'] as const;
```

Comment:
Why separate file? This could just go into `carousel-utils.ts`.

Reply:
Done